packages/coinjoin/src/client/Account.ts

```diff
@@ -1,11 +1,28 @@
 import { Network } from '@trezor/utxo-lib';
 
-import { getScriptPubKeyFromAddress } from '../utils/coordinatorUtils';
+import { getScriptPubKeyFromAddress, prefixScriptPubKey } from '../utils/coordinatorUtils';
 import { AllowedScriptTypes } from '../types/coordinator';
 import { RawLiquidityClue } from '../types/middleware';
 import { RegisterAccountParams } from '../types';
 import { AccountUtxo, AccountAddress } from '../types/account';
 
+const enhanceAccountUtxo = (
+    utxos: Omit<AccountUtxo, 'scriptPubKey'>[],
+    network: Network,
+    scriptType: AllowedScriptTypes,
+): AccountUtxo[] =>
+    utxos.flatMap(address => ({
+        path: address.path,
+        address: address.address,
+        scriptPubKey: prefixScriptPubKey(
+            getScriptPubKeyFromAddress(address.address, network, scriptType),
+            true,
+        ),
+        outpoint: address.outpoint,
+        amount: address.amount,
+        anonymityLevel: address.anonymityLevel,
+    }));
```

Comment:
I would suggest to rename the local element from `address` to `utxo` or avoid the naming completely like this: 
```suggestion
    utxos.flatMap(({ path, address, outpoint, amount, anonymityLevel }) => ({
      path,
      address,
      scriptPubKey: prefixScriptPubKey(
        getScriptPubKeyFromAddress(address, network, scriptType),
        true,
      ),
      outpoint,
      amount,
      anonymityLevel,
    }));
```

Reply:
btw, here is nothing to flatten, right? `map` should be used instead of `flatMap`

Reply:
[63276a3](https://github.com/trezor/trezor-suite/pull/7405/commits/63276a384a32862b01a17af7f3c374d2fefe88c9)

i did the same with `enhanceAccountAddresses`